spec/system/header_nav_spec.rb

```diff
@@ -81,10 +81,18 @@
                                                  }.from('Your list').to('Manage users')
       end
 
-      it 'shows the "Performance tracking" link and it can be followed' do
-        expect { click_link('Performance tracking') }.to change {
-          page.first('.govuk-heading-xl').text
-        }.from('Your list').to('Performance tracking')
+      it 'does not show the "Performance tracking" link' do
+        expect(page).not_to have_link('Performance tracking')
+      end
+
+      context 'when user managers are logged in as supervisors' do
```

Comment:
This test makes it sounds like the user manager can change their role perhaps 'when user manager is supervisor' ?